cibuildwheel/oci_container.py

```diff
@@ -155,8 +176,13 @@ class OCIContainer:
         >>> from cibuildwheel.oci_container import *  # NOQA
         >>> from cibuildwheel.options import _get_pinned_container_images
         >>> image = _get_pinned_container_images()['x86_64']['manylinux2014']
+        >>> try:
+        >>>     oci_platform = OCIPlatform.current()
+        >>> except EnvironmentError as ex:
```

Comment:
nit- it was OSError above (though it is equivalent)
```suggestion
        >>> except OSError as ex:
```